### ISSUE TITLE
DREAMING BIG: A stand-alone year-end Mapbox map with all the 2017 town halls?

### ISSUE THREAD

**Issue by nathanmwilliams** [townhallproject/townHallProject#229]
Can we have a "2017 wrap-up" version of our map with all the 2017 town halls (plus Empty Chair and Adopt-A-Districts?). Would not need any other event types.

**Comment by meganrm** [comment]
yeah! We might want to try to find a way to animate things so that it's not frustratingly slow as it loads

**Comment by nathanmwilliams** [comment]
Yeah, it can be as static as we need it to be to load/work efficiently.

**Comment by nathanmwilliams** [comment]
Perhaps more realistic timeline for this is August--right as we move from official MOC events to campaign events only.

**Comment by meganrm** [comment]
I was thinking this could be a part of a "explore our data" page.
